tests/foreman/api/test_contentmanagement.py

```diff
@@ -137,6 +138,39 @@ def test_positive_sync_kickstart_repo(self, module_manifest_org, default_sat):
         assert rh_repo.content_counts['package_group'] > 0
         assert rh_repo.content_counts['rpm'] > 0
 
+    @pytest.mark.parametrize('distro', ['rhel6', 'rhel7', 'rhel8', 'rhel9'])
```

Comment:
I'd replace the static rhel version list by:
```
[f'rhel{ver}' for ver in settings.supportability.content_hosts.rhel.versions]
```
to automatically expand to everything we support in the current branch (sat version)
you might need to rebase since this got in just now with:
https://github.com/SatelliteQE/robottelo/pull/9353/files

Reply:
ACK, I will rebase.